handlers/proxy_logging.go

```diff
@@ -0,0 +1,53 @@
+package handlers
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"github.com/gorilla/mux"
+)
+
+var (
+	validLogLevels = []string{"off", "trace", "debug", "info", "warning", "error", "critical"}
```

Comment:
Perhaps the "handler" layer should not have information the list of available levels.
I think that could be part of the business package.

To give an example, if you look into the "handlers/istio_config.go" you'll see that the there is a criteria parsed and built in the "business" package. 

Then if you need to maintain this list, that would be located under the "business" layer.

My suggerstion would be to move the "validLogLevels" and "isValidLogLevel" to the business layer and just invoke in the "handler" a business.IsValidLogLevel.

It's just code style, logic itself is ok.

Reply:
Sure that makes sense to me to move it there